options.c

```diff
@@ -4665,6 +4665,16 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 		.category = FIO_OPT_C_IO,
 		.group	= FIO_OPT_G_IO_BUF,
 	},
+	{
```

Comment:
Remember to increase FIO_SERVER_VER when you make a change that makes the on-wire protocol incompatible.